config/prod/neo4j2owl_config_schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "neo4j2owl-config schema",
+  "description": "A JSON schema for documenting and validating the neo4j2owl configuration YAML.",
+  "name": "neo4j2owl_config_schema",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "semantic_tag": {
```

Comment:
@bvarner-ebi - please review description text for semantic tag and its properties.  This describes the content in an array under the 'neo_node_labelling' property.  The text needs to communicate content and intent clearly.